package.json

```diff
@@ -36,17 +37,18 @@
     "flow-bin": "0.48.0",
     "mocha-steps": "1.1.0",
```

Comment:
The version of enzyme-adapter-react-16 can also be bumped with react scripts

Reply:
I am planning another set of changes for updating everything that we can.  This one is really intended just to fix npm audit.  I'll try it with a newer version tho.